views/mailinglistwidget.ejs

```diff
@@ -0,0 +1,44 @@
+<ul id="groups" class="group-list">
```

Comment:
At some point could you clean up the indentations in this file? It's kind of a mess and difficult to follow the blocks :)